### PR TITLE
feat(switchTxType): add new component #124

### PR DESCRIPTION
**Pull request by escapedcat**

### CODE REVIEW

**Comment by cstenglein** [comment]
Cool idea with the extra component! :)

**Review by cstenglein — COMMENTED** [review]
Just a small thing :)

**Comment by cstenglein** [comment]
Thank you @escapedcat looks awesome! :)

File: src/components/Shared/SwitchTxType/SwitchTxType.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import { useState } from "react";
+import { useTranslation } from "react-i18next";
+import type { FC } from "react";
+
+export enum TxType {
+  onChain = "ONCHAIN",
```
[the remark this answers is not in the document]
Comment (escapedcat) [reply src/components/Shared/SwitchTxType/SwitchTxType.tsx]: ✅

**Comment by cstenglein** [comment]
@escapedcat thank you! :)

**Pull request merged**